### ISSUE TITLE
[Suggestion] GUI player input

### ISSUE THREAD

**Issue by Matthysse** [Blackixx/BossShopPro#67]
Hello!

 It would be nice to have the ability to set, when right click an icon in a GUI, that there is an avil GUI or sign GUI so that the player can write something that we can integrate in a command, for example:
If we sell /setwarps, we could create an item in the GUI which would be named "Set a warp". When the player right click on it, the anvil or sign gui would open and the player would be able to set the name of his warp, then this input could be used in the command we set. It would be so nice, so the player doesn't have to type a long command.

**Comment by Blackixx** [comment]
Player input is already supported. For more info, take a look at https://www.spigotmc.org/wiki/bossshoppro-configuration-advanced-options/. The player has to enter the input into the chat though, an anvil or sign GUI is not supported.

**Comment by Matthysse** [comment]
Yeah that's why I opened this issue, to see the integration of Anvil GUIs, would make the GUIs look more "professional"
